elftools/elf/elffile.py

```diff
@@ -12,6 +12,8 @@
 
 try:
     import resource
+    if "getpagesize" not in dir(resource):
```

Comment:
I think it would be more Pythonic to check for AttributeError along with ImportError in the `except` clause. What do you think?